pandas/tseries/tests/test_timedeltas.py

```diff
@@ -1530,14 +1530,19 @@ def test_ops_error_str(self):
                 l != r
 
     def test_map(self):
-
         rng = timedelta_range('1 day', periods=10)
 
         f = lambda x: x.days
         result = rng.map(f)
         exp = Int64Index([f(x) for x in rng])
         tm.assert_index_equal(result, exp)
 
+        s = Series(exp, index=rng)
```

Comment:
sam as above (you might wish to move these tests to test_datetimelike to avoid duplication)